src/main/java/org/homeschoolpebt/app/cli/TransmitterCommands.java

```diff
@@ -102,16 +114,24 @@ private void zipFiles(Map<String, Submission> appIdToSubmission, String zipFileN
         String subfolder = createSubfolderName(submission, transmission);
         try {
           if ("pebt".equals(submission.getFlow())) {
-            // generate applicant summary
-            byte[] file = pdfService.getFilledOutPDF(submission);
-
-            String fileName = pdfService.generatePdfName(submission);
-            zos.putNextEntry(new ZipEntry(subfolder));
-            ZipEntry entry = new ZipEntry(subfolder + fileName);
-            entry.setSize(file.length);
-            zos.putNextEntry(entry);
-            zos.write(file);
-            zos.closeEntry();
+            // delay 7 days if there aren't any uploaded docs associated with this application
+            Date submittedAt = submission.getSubmittedAt();
+            Date sevenDaysAgo = new Date(System.currentTimeMillis() - SEVEN_DAYS_IN_MS);
+            boolean hasLaterDocs = appIdToLaterDoc.containsKey(appNumber);
+
+            Map<String, Object> inputData = submission.getInputData();
+            if (submittedAt.before(sevenDaysAgo) || hasLaterDocs || UPLOAD_DOCS.stream().anyMatch(k -> inputData.get(k) != null && !inputData.get(k).equals(emptyList()))) {
```

Comment:
The way we should be checking for the lack of skipped Doc Upload sections is via this method:

```java
SubmissionUtilities.getMissingDocUploads(submission).isEmpty()
```